README.md

```diff
@@ -141,17 +141,19 @@ The STL uses boost-math headers to provide P0226R1 Mathematical Special Function
 acquire this dependency.
 
 1. Install Visual Studio 2019 16.6 Preview 2 or later.
-2. Invoke `git clone https://github.com/microsoft/vcpkg`
-3. Invoke `cd vcpkg`
-4. Invoke `.\bootstrap-vcpkg.bat`
-5. Assuming you are targeting x86 and x64, invoke `.\vcpkg.exe install boost-math:x86-windows boost-math:x64-windows`
+2. Change directories to a location where you'd like a clone of this STL repository.
+3. Invoke `git clone https://github.com/microsoft/STL`
+4. Invoke `cd STL`
+5. Invoke `git submodule update --init -- vcpkg/`

```

Comment:
Is `--` necessary? We know that `vcpkg` doesn't look like an option.

Reply:
Didn't seem to work without it when I tried.

Reply:
Yes this works => git submodule update --init vcpkg
I'll update accordingly.

Reply:
Is it worth updating Issue #805 to also include the submodule update for vcpkg then?  It may not be as much of an issue since the vcpkg repo is much smaller than llvm-project.

Reply:
That's an excellent idea. While vcpkg is small, it's always possible for network congestion, wifi interference, or server issues to make things slow.

Reply:
Great, I've updated #805, linking this PR and #761 as well.